src/sidebar/components/test/selection-tabs-test.js

```diff
@@ -87,28 +96,108 @@ describe('selectionTabs', function () {
+
+      var elem = util.createDirective(document, 'selectionTabs', {
+        selectedTab: 'note',
+        totalAnnotations: '10',
+        totalNotes: 0,
+      });
+      var unavailableMsg = elem[0].querySelector('.annotation-unavailable-message__label');
+      var unavailableTutorial = elem[0].querySelector('.annotation-unavailable-message__tutorial');
+      var noteIcon = unavailableTutorial.querySelector('i');
+
+      assert.include(unavailableMsg.textContent, 'There are no page notes in this group.');
+      assert.include(unavailableTutorial.textContent, 'Create one by clicking the');
+      assert.isTrue(noteIcon.classList.contains('h-icon-note'));
+    });
+
+    it('should display the longer version of the no annotations message when there are no annotations', function () {
+      fakeSession.state.preferences.show_sidebar_tutorial = false;
+      fakeSettings.enableExperimentalNewNoteButton = false;
+
+      var elem = util.createDirective(document, 'selectionTabs', {
+        selectedTab: 'annotation',
+        totalAnnotations: 0,
+        totalNotes: '10',
+      });
+      var unavailableMsg = elem[0].querySelector('.annotation-unavailable-message__label');
+      var unavailableTutorial = elem[0].querySelector('.annotation-unavailable-message__tutorial');
+      var noteIcon = unavailableTutorial.querySelector('i');
+
+      assert.include(unavailableMsg.textContent, 'There are no annotations in this group.');
+      assert.include(unavailableTutorial.textContent, 'Create one by selecting some text and clicking the');
+      assert.isTrue(noteIcon.classList.contains('h-icon-annotate'));
+    });
+
+    context('when the sidebar tutorial is displayed', function () {
+      fakeSession.state.preferences.show_sidebar_tutorial = true;
+
+      it('should display the shorter version of the no notes message when there are no notes', function () {
+        var elem = util.createDirective(document, 'selectionTabs', {
+          selectedTab: 'note',
+          totalAnnotations: '10',
+          totalNotes: 0,
+        });
+        var msg = elem[0].querySelector('.annotation-unavailable-message__label');
+
+        assert.include(msg.textContent, 'There are no page notes in this group.');
+        assert.notInclude(msg.textContent, 'Create one by clicking the');
+        assert.notInclude(msg.textContent, 'Create one by selecting some text and clicking the');
+      });
+
+      it('should display the shorter version of the no annotations message when there are no annotations', function () {
+        var elem = util.createDirective(document, 'selectionTabs', {
+          selectedTab: 'annotation',
+          totalAnnotations: 0,
+          totalNotes: '10',
+        });
+        var msg = elem[0].querySelector('.annotation-unavailable-message__label');
+
+        assert.include(msg.textContent, 'There are no annotations in this group.');
+        assert.notInclude(msg.textContent, 'Create one by clicking the');
```

Comment:
It looks like this line is a mistake as well, since "Create one by clicking the" is only shown in the notes tab.

Just as a suggestion, it might save a few lines and make the tests more readable to extract out the logic for querying the DOM into a helper function which returns some value to indicate the displayed message that you can then test for. Alternatively if the template "logic" is very simple (eg. hiding/showing different elements depending on some controller methods which return a boolean) you can might be able to get away with just testing the controller method's output in different cases.

Reply:
Discussed this on Slack. This is a check to make sure that the annotations-related message is not showing.